src/content-tooltip/constants.js

```diff
@@ -8,3 +8,19 @@ export const POSITION_STORAGE_NAME = 'memex_link_position'
 
 export const INFO_URL =
     'https://worldbrain.helprace.com/i62-feature-memex-links-highlight-any-text-and-create-a-link-to-it'
+
+export const KEYBOARDSHORTCUTS_STORAGE_NAME = 'memex_keyboardshortcuts'
```

Comment:
It would be great to add an entry to our local storage keys docs for this in `docs/Local-Storage-Keys.md` (https://github.com/WorldBrain/Memex/blob/develop/docs/Local-Storage-Keys.md). Then when other people come across this in the future, they can look it up and see what it does without going through all the code.